XenOvfApi/app.config

```diff
@@ -67,8 +67,8 @@
    <setting name="xenBootOptions" serializeAs="String">
     <value>BIOS order</value>
    </setting>
-   <setting name="xenBootOrder" serializeAs="String">
-    <value>dc</value>
+   <setting name="xenBootParams" serializeAs="String">
+    <value>order=dc;</value>

```

Comment:
Hi  @kc284,
It occurred to me that this change might break the logic if there were some persistent config in the customers' environment.
I just went through the modification history of the file and found no structural changes.
So for my CP ticket, I would remain xenBootOrder as it was, and create a new xenBootMode node to implement my job, but we may suffer from changing it again in future.
Then what should we do if we really want to change the existing config?